app/smc/prj.conf

```diff
@@ -35,3 +35,5 @@ CONFIG_BOOT_BANNER=n
 CONFIG_TT_BOOT_BANNER=y
 CONFIG_TT_BOOT_BANNER_GIT_VERSION=y
 CONFIG_TT_BOOT_BANNER_SDK_VERSION=y
+
+CONFIG_CLOCK_CONTROL=y
```

Comment:
This should probably be moved to the board `tt_blackhole_tt_blackhole_smc_defconfig`